src/main/resources/view/MainWindow.fxml

```diff
@@ -3,15 +3,17 @@
 <?import java.net.URL?>
 <?import javafx.geometry.Insets?>
 <?import javafx.scene.Scene?>
+<?import javafx.scene.control.Button?>
 <?import javafx.scene.control.Menu?>
 <?import javafx.scene.control.MenuBar?>
 <?import javafx.scene.control.MenuItem?>
 <?import javafx.scene.image.Image?>
+<?import javafx.scene.layout.HBox?>
 <?import javafx.scene.layout.StackPane?>
 <?import javafx.scene.layout.VBox?>
+<?import javafx.stage.Stage?>
 
-<fx:root type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1"
-         title="WIFE" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
+<fx:root alwaysOnTop="true" maxHeight="900.0" maxWidth="1200.0" minHeight="650.0" minWidth="700.0" onCloseRequest="#handleExit" title="WIFE" type="javafx.stage.Stage" width="680.0" xmlns="http://javafx.com/javafx/19" xmlns:fx="http://javafx.com/fxml/1">
   <icons>
     <Image url="@/images/address_book_32.png" />
```

Comment:
Change this to WIFE

Reply:
i think this is just the icon name - because it's an address book
![image](https://user-images.githubusercontent.com/35266074/227493462-9a2f96a1-7a68-4940-9ab4-8a0b8bbe53ee.png)